### ISSUE TITLE
Wrong stacktrace when using assert

### ISSUE THREAD

**Issue by ArnaudBarre** [oven-sh/bun#7854]
### What version of Bun is running?

1.0.20+09d51486e

### What platform is your computer?

Darwin 23.1.0 arm64 arm

### What steps can reproduce the bug?

Run
```
import assert from "node:assert";

if (true) {
  assert(false);
}
```

### What is the expected behavior?

Error pointer to line 4

### What do you see instead?

Error point to line 3

**Comment by Jarred-Sumner** [comment]
our sourcemaps are not in a good place right now

it's on our TODO list to address

**Comment by paperdave** [comment]
This will be resolved in Bun 1.1.13
